### DISCUSSION TITLE
Is luaunit still maintained?

### DISCUSSION THREAD

**Discussion by Ismoh** [bluebird75/luaunit#144] [Q&A]
Is it dead or alive?

**Comment by bluebird75** [comment]
What makes you think it's dead ? It's alive and fully functional!

**Reply by Ismoh** [reply]
I was tired, didnt want to bother you.
I wasnt able to find assertTableContains in the docs, 
code coverage is at 0% since last pr merge
and there is a really old PR open and uncomment.

That are more or less the reasons I was afraid luaunit isnt maintained anymore.

**Reply by bluebird75** [reply]
I had a fresh look and it looks abondoned indeed.

I'll fix it. It's true that my attention has been elsewhere those last few months.

Until I understand why readthedoc does not generate the latest documentation, you can find assertTableContains here  : 

https://github.com/bluebird75/luaunit/blob/master/doc/index.rst#table-assertions

**Comment by Ismoh** [comment]
Thanks a lot for having a look into it! Surely you also have other projects.
I just want luaunit not to look like its abandoned.

(maybe you can an answer here, so that I can mark it as answered)
